packages/scandipwa/src/component/MyAccountOverlay/MyAccountOverlay.style.scss

```diff
@@ -158,6 +158,13 @@
             padding-block-start: 14px;
             padding-inline: 14px;
         }
+
+        &_state_forgotPassword {
+            .Button.Button_likeLink {
```

Comment:
Do we need to increase specificity here with two class names?

Reply:
Yes, you need.

But actually this is due to too general selector used in another part of code. I added more classes to elements I want to style + got rid of too general `.Button` selector => 2 class names are not needed any more.